pkg/api/vlabs/types.go

```diff
@@ -390,6 +390,7 @@ type MasterProfile struct {
 	AgentSubnet              string            `json:"agentSubnet,omitempty"`
 	AvailabilityZones        []string          `json:"availabilityZones,omitempty"`
 	SinglePlacementGroup     *bool             `json:"singlePlacementGroup,omitempty"`
+	PlatformFaultDomainCount *int              `json:"platformFaultDomainCount,omitEmpty"`
```

Comment:
In order to compose both vlabs and "unversioned" data types we need these property additions to `MasterProfile` and `AgentPoolProfile` to be in `pkg/api/convertertoapi.go` and `pkg/api/converterfromapi.go`

Reply:
done in a22edcb